docs/articles/new-expensify/get-paid-back/Distance-Requests.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Distance Requests
+description: How to create a distance request and request reimbursement for mileage
+---
+<!-- The lines above are required by Jekyll to process the .md file -->
+
+# Overview
+
+Expensify allows you to request reimbursement for mileage by creating a distance request from a map. You can send a distance request in Expensify's mobile, desktop, or web app.
+
+
+# How to create and send a distance request
```

Comment:
NAB: Potential improvement, remove the "and send" part since it's a bit redundant. You can't create one without sending it too.


# How to create a distance request

Reply:
I personally think "and send" is helpful. I know that you can't do one without the other in NewDot, but users are used to thinking about creating and sending separately so they may look for a separate "Send" section if it's not called out.

Reply:
Ok that's fair.